web/tests/upload.spec.ts

```diff
@@ -0,0 +1,26 @@
+// Test uploading a file in an import box.
+import { test, expect, Browser, Page } from '@playwright/test';
+import { Workspace, TableState } from './lynxkite';
+import { resolve } from 'path';
+
+test.only('can upload and import a simple CSV', async ({ page }) => {
+  const workspace = await Workspace.empty(page);
+  await workspace.addBox({ id: 'imp', name: 'Import CSV', x: 100, y: 100 });
+  const box = await workspace.openBoxEditor('imp');
+  const fileChooserPromise = page.waitForEvent('filechooser');
+  const uploadButton = box.getParameter('filename', '[drop-tooltip="Upload"]');
+  await uploadButton.click();
+  const fileChooser = await fileChooserPromise;
+  await fileChooser.setFiles(resolve('tests/data/upload_test.csv'));
```

Comment:
From https://playwright.dev/docs/api/class-filechooser. Now it really uploads a file. Surprisingly, the original `upload-test.js` does no such thing! It just uses the file that's already on the disk. (This functionality is also tested by `plot.spec.ts` so nothing is lost by fixing this.)

Reply:
What is a difference between uploading a file and "using the file that's already on the disk"?

Reply:
> What is a difference between uploading a file and "using the file that's already on the disk"?

I would hope that the code tells you the difference! To upload a file we click the upload button and select a file. (This code.) To use a file from disk we just enter the filename. (The original code.)